go/app/infra.go

```diff
@@ -45,5 +126,23 @@ func (i *itemRepository) Insert(ctx context.Context, item *Item) error {
 func StoreImage(fileName string, image []byte) error {
 	// STEP 4-4: add an implementation to store an image
 
+	imageDir := "images"
+
+	if _, err := os.Stat(imageDir); os.IsNotExist(err) {
+		err := os.MkdirAll(imageDir, os.ModePerm)
```

Comment:
(コメント) ModePermはfile permissonが777になってしまうのですが、一般的に777にするのはよろしくないこともある (特に今回のDBのファイルなどは) ので、755にしておくのが無難かもしれないです。
```suggestion
		err := os.MkdirAll(imageDir, 755)
```